### PR TITLE
Pick one module of each type

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

**Comment by mehalshah** [comment]
Instead of the old multi-select for overriding your evaluation, there's now two lists of content and practice modules. You need to pick one of each.

image

File: dashboard/test/controllers/plc/enrollment_evaluations_controller_test.rb [hunk 1]
```diff
@@ -12,36 +12,43 @@ class Plc::EnrollmentEvaluationsControllerTest < ActionController::TestCase
     #the same for everyone. Not all users taking an examination will answer the exact same questions.
     @course = create(:plc_course)
     @course_unit = create(:plc_course_unit, plc_course: @course)
-    @module1 = create(:plc_learning_module, name: 'Getting thrown off cliffs', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE)
-    @module2 = create(:plc_learning_module, name: 'Advanced Ornithology', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
-    @module3 = create(:plc_learning_module, name: 'Answering questions honestly', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE)
-    @module4 = create(:plc_learning_module, name: 'Admitting ignorance', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
 
+    @module_cliffs = create(:plc_learning_module, name: 'Getting thrown off cliffs', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
```
Comment (aoby) [inline dashboard/test/controllers/plc/enrollment_evaluations_controller_test.rb:16]: Is this learning about surviving the throw, or it just signing up to actually be thrown? ;)

Great test module names.

File: dashboard/app/models/plc/learning_module.rb [hunk 2]
```diff
@@ -35,6 +35,8 @@ class Plc::LearningModule < ActiveRecord::Base
   attr_readonly :plc_course_unit_id
 
   scope :required, -> { where(module_type: REQUIRED_MODULE)}
+  scope :content, -> { where(module_type: CONTENT_MODULE)}
+  scope :practice, -> { where(module_type: PRACTICE_MODULE)}
```
Comment (aoby) [inline dashboard/app/models/plc/learning_module.rb]: nit: inconsistent spacing at beginning and end of block.

**Comment by aoby** [comment]
lgtm

File: code-studio/src/css/plc/preview_evaluation.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.preview_evaluation {
+  display: flex;
```
Comment (joshlory) [inline code-studio/src/css/plc/preview_evaluation.scss:2]: Have you checked this page in IE?  It looks like their `display: flex;` support may be spotty: http://caniuse.com/#feat=flexbox

Comment (mehalshah) [inline code-studio/src/css/plc/preview_evaluation.scss:2]: Flex is supported enough on IE 11. On IE 9/10 it degrades to

image

So still functional

File: dashboard/test/controllers/plc/enrollment_evaluations_controller_test.rb (hunk 1 above)
Comment (mehalshah) [inline dashboard/test/controllers/plc/enrollment_evaluations_controller_test.rb:16]: I don't know that. Augghhh!

**Pull request merged**